presto-orc/src/main/java/com/facebook/presto/orc/writer/SliceDictionaryColumnWriter.java

```diff
@@ -74,7 +74,7 @@ public SliceDictionaryColumnWriter(
         super(column, type, columnWriterOptions, dwrfEncryptor, orcEncoding, metadataWriter);
         this.dictionaryDataStream = new ByteArrayOutputStream(columnWriterOptions, dwrfEncryptor, Stream.StreamKind.DICTIONARY_DATA);
         this.dictionaryLengthStream = createLengthOutputStream(columnWriterOptions, dwrfEncryptor, orcEncoding);
-        this.stringStatisticsLimitInBytes = toIntExact(columnWriterOptions.getStringStatisticsLimit().toBytes());
+        this.stringStatisticsLimitInBytes = columnWriterOptions.getStringStatisticsLimit();
```

Comment:
Imho this was unnecessary

Reply:
This is being called at two places now, and the caller only expects a integer size, so decided to change it so that both of them can reuse this.